files/image_config/monit/memory_checker

```diff
@@ -45,20 +69,104 @@ def get_command_result(command):
     command_stdout = ""
 
     try:
-        proc_instance = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+        proc_instance = subprocess.Popen(command.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE,
```

Comment:
>.split() [](http://example.com/codeflow?start=48&length=8)

Suggest not to use split and pass command as an array instead

Reply:
fixed.